extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -467,12 +467,19 @@ public void setAssertion(boolean assertion) {
         public static class Provider {
 
             /**
-             * The CredentialsProvider name, which should only be set if more than one CredentialsProvider is
+             * The CredentialsProvider bean name, which should only be set if more than one CredentialsProvider is
              * registered
              */
             @ConfigItem
             public Optional<String> name = Optional.empty();
 
+            /**
+             * The CredentialsProvider keyring name, which should always be set when using a CredentialsProvider
```

Comment:
@ryandens It should remain optional, if my provider just reads a secret from the local database, there should be no need to set this property. Also a missing `n` at the end of the doc

Reply:
Please update the message to explain when it must be set but let users know they are not required to set it